transactions.md

```diff
@@ -191,3 +192,38 @@ This transaction spends the [`unvault_tx`](unvault_tx) `output[0]` to the EDV by
 - outputs[0]:
     - value: `<unvault_tx outputs[0] value - fees>`
     - scriptPubkey: `0x00 SHA256(<EDV_script>)`
+
+
+## bypass_tx
+
+Bypass txs are used in the case where stakeholders need immediate access to funds. Bypass 
+txs avoid the controls on expenses set by stakeholders and enforced by watchtowers. 
+
+A Bypass transaction spends the [`deposit_tx`](deposit_tx) output by the `N`-of-`N` path and 
+pays to a arbitrary addresses.
+
+- version: 2
+- locktime: 0
+
+#### IN
+
+- count: 1
+- inputs[0]:
+    - txid: `<deposit_tx txid>`
+    - vout: `<deposit_tx vout>`
+    - sequence: `0xfffffffd`
+    - scriptSig: `<empty>`
+    - witness: `satisfy(vault_descriptor)`
```

Comment:
Arg we should rename this descriptor to `deposit` too (not necessarily in this PR)